Cargo.toml

```diff
@@ -11,21 +12,20 @@ members = [
     "pallets/feature-control",
     "pallets/committee-management",
     "pallets/support",
+    "pallets/vk-storage",
     "primitives",
     "rate-limiter",
+    "runtime-interfaces",
 ]
 
 exclude = [
```

Comment:
Nice :) So now `exclude` resembles crates that should be moved to the separate repository (in the future).

CC @kostekIV